docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,142 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _vertiq_testing__guide:
+
+########################################
+Getting Started with Vertiq Testing Tool
+########################################
+
+*****
+About
+*****
+
+Vertiq Testing Tool is a supplemental application to :ref:`IQ Control Center <control_center_start_guide>`. 
+It provides a streamlined interface to control your module using ESC, Voltage, Velocity, and PWM inputs.
+
+
+********
+Download
+********
+Currently, Vertiq Testing Tool is only available on Windows.
+You can download Vertiq Testing Tool on our `Support Page <https://www.vertiq.co/support>`_.
+
+
+**********************
+Hardware Configuration
+**********************
+Please follow the 'Connection with a Computer' and 'Wiring' sections of the :ref:`Control Center documentation <connection_guide>` to connect your module to your computer.
+
+
+********
+Overview
+********
+Once downloaded, extract and run the Vertiq Testing Tool executable. When running this application for the first time, 
+you may encounter a Windows Defender SmartScreen warning.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/windows_defender.png
+
+Click on 'More info'.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/windows_defender_more_info.png
+
+Then click 'Run anyway'.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/welcome_screen.png
+
+Once the application loads, you will be presented with the welcome screen. Follow the instructions on this screen to connect your module.
+
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input.png
+
+Once connected, you should see the ESC Input tab as shown above. The Control Panel contains the same parameters necessary to configure your module for use with a flight controller as in this :ref:`Control Center example <flight_controller_config_with_control_center>`.
+
+#. The :goldenrod:`Tabs` section provides different widgets for controlling your module. ESC Input provides configuration parameters necessary for integrating properly with a flight controller as well as simulating flight controller throttle commands via the IQUART raw value message. 
```

Comment:
It's nice that you describe a bit what the ESC Input tab is for at a high level here. it may be good to give a brief, one sentence thing like that for each tab here. Like:

- The`Tabs` section provides different widgets for controlling your module:
       -The ESC Input provides....
       -The Voltage Input tab provides.....
       -The Velocity Input tab provides......
       -The PWM Input tab provides......

Can be super simple one sentence things, the basic high level over view is just nice